companies/eastbankclub.json

```diff
@@ -0,0 +1,23 @@
+{
+    "slug": "eastbankclub",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "health"
+    ],
+    "name": "East Bank Club The Fitness Factory GmbH",
+    "runs": [
+        "SUPERFIT"
+    ],
+    "address": "Postfach 410108\n12111 Berlin\nDeutschland",
+    "phone": "+49 30 688381768",
```

Comment:
At the bottom of the privacy policy, a phone number specific to the data protection officer is listed. We should use that one instead.